### PR TITLE
NO-JIRA Fix capitalization of "Qpid Dispatch Router"

### PR DESCRIPTION
**Pull request by jiridanek**
When talking about "router network", I was not sure if router should be capitalized. I decided to make it so, but that can be changed...

CC @pwright

### CODE REVIEW

**Review by pwright — APPROVED** [review]
capitalisation is good, the "Dispatch Router", but a "router" seems to be standard through out https://qpid.apache.org/releases/qpid-dispatch-1.12.0/user-guide/index.html

File: console/react/src/connect/connectPage.js [hunk 1]
```diff
@@ -82,7 +82,7 @@ class ConnectPage extends React.Component {
           </TextContent>
           <TextContent>
             <Text component="h2" className="connect-description">
-              The console displays information about a qpid dispatch router network. It
+              The console displays information about a Qpid Dispatch Router network. It
```
Comment (jiridanek) [inline console/react/src/connect/connectPage.js:85]: @pwright This is where I wasn't sure about "router". Should the sentence be

* displays information about a Qpid Dispatch *R*outer network

or

* displays information about a Qpid Dispatch *r*outer network?

I am currently thinking that lowercase r is better choice here.

Comment (pwright) [reply console/react/src/connect/connectPage.js:85]: to be consistent with the docs you'd use a capital R here.
An alternative would be to reword to:

* displays information about a router network

which, to my eye, looks better. No need to explicitly state the 'long name' everywhere. wdyt?

Comment (jiridanek) [reply console/react/src/connect/connectPage.js:85]: Thanks, I'll go with capital R then.

**Pull request merged**